app/frontend/components/common/balance.js

```diff
@@ -40,7 +41,32 @@ const Balance = ({balance, reloadWalletInfo, conversionRates}) =>
         'Refresh Balance'
       )
     ),
-    conversionRates && h(Conversions, {balance, conversionRates})
+    conversionRates && h(Conversions, {balance, conversionRates}),
+    h(
+      'div',
+      {class: 'buy-ada'},
+      h(
+        'a',
+        {
+          class: 'button primary outline buy-ada-link',
+          href: 'https://exchange.adalite.io/',
+          target: '_blank',
+        },
+        'Buy ADA'
+      ),
+      h(
+        'span',
+        {
+          class: 'buy-ada-logo',
+        },
+        'Powered by',
+        h('img', {
+          class: 'buy-ada-icon',
+          src: LOGO_COINSWITCH,
```

Comment:
I would directly reference the asset url here. Other parts of the code reference the assets directly as well

Reply:
@refi93 Thanks, we discussed this in person with @ladomyriak  👍

Reply:
Fixed